torch/__init__.py

```diff
@@ -268,6 +268,8 @@ def typename(o):
 
     return module + class_name
 
+def is_bf16_support():
+    return torch.cuda.get_device_properties(torch.cuda.current_device()).major >= 8
```

Comment:
This also should include check for cuda 11 (theoretically, you can jit-compile pytorch cuda 10 version to run on amperes)

Reply:
ok